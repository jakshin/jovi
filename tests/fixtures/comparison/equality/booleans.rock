(Using the equality comparison operator on booleans)

Rightness is right
Wrongness is wrong

Say "booleans equal themselves (duh)"
Rightness is right
Wrongness is wrong
say Rightness is Rightness
say Wrongness is Wrongness
say Rightness is Wrongness
say true is true
say false is false
say true is false

(We test booleans against numbers and strings in numbers.rock and strings.rock)
