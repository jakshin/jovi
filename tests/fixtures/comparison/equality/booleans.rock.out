booleans equal themselves (duh)
true
true
false
true
true
false
